resource-manager/src/main/scala/org/apache/spark/scheduler/cluster/ytsaurus/YTsaurusOperationManager.scala

```diff
@@ -168,6 +168,10 @@ private[spark] class YTsaurusOperationManager(
       appArgs.mainClass
     ) ++ additionalArgs ++ appArgs.driverArgs).mkString(" ")
 
+    if (conf.getBoolean("spark.ytsaurus.redirect.stdout.to.stderr", defaultValue = false)) {
```

Comment:
You should define config option here: resource-manager/src/main/scala/org/apache/spark/deploy/ytsaurus/Config.scala